std_msgs/Header header
std_msgs/Int32 num_positions
std_msgs/Int32 position
std_msgs/Float32 position_width_radians
std_msgs/Float32 detent_strength_unit
std_msgs/Float32 endstop_strength_unit
std_msgs/Float32 snap_point
std_msgs/String text
std_msgs/Float32 tcp_force
